src/components/manage/Controlpanels/Groups/GroupsControlpanel.jsx

```diff
@@ -108,11 +109,15 @@ class GroupsControlpanel extends Component {
   }
 
   fetchData = async () => {
+    await this.props.getControlpanel('usergroup');
     await this.props.listRoles();
-    await this.props.listGroups();
-    this.setState({
-      groupEntries: this.props.groups,
-    });
+    if (!this.props.many_groups) {
+      await this.props.listRoles();
```

Comment:
I think one `this.props.listRoles()` is enough and is OK also for many_groups

Reply:
Done @ksuess , I have also added cypress for relevant scenarios.